### PR TITLE
 chore(init-templates): fix import of init templates

### PR DESCRIPTION
**Pull request by lune-sta**
TypeScript init-templates are not imported `cdk.Duration`.
Fixed to avoid confusion for new writing CDK.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by mrgrain** [comment]
Hi @lune-sta thanks for your contribution. Please make sure you have working tests and added or updated tests where appropriate.

I support the motivation behind this change. Currently the created template would run, with your change it won't anymore since `Duration` is unused. I think the right way to solve this will be to move the `core` imports to `import * as cdk from 'aws-cdk-lib'`. We use that elsewhere in the templates already and [recommend it for contributions](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#recommendations) to the libraries as well.

**Comment by lune-sta** [comment]
@mrgrain Thank you.
I modified it to use `import * as cdk from 'aws-cdk-lib'`.
The test is passing locally, let me check CI.
```
Test Suites: 59 passed, 59 total
Tests:       510 passed, 510 total
Snapshots:   0 total
Time:        18.386 s, estimated 22 s
Ran all test suites.
Tests successful.
```

**Review by mrgrain — CHANGES_REQUESTED** [review]

**Comment by mrgrain** [comment]
>The test is passing locally, let me check CI.

Looks like this was blip in CI.

File: packages/aws-cdk/lib/init-templates/app/typescript/lib/%name%-stack.template.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// import * as cdk from 'aws-cdk-lib'
 import { Stack, StackProps } from 'aws-cdk-lib';
```
[the remark this answers is not in the document]
Comment (lune-sta) [reply packages/aws-cdk/lib/init-templates/app/typescript/lib/%name%-stack.template.ts]: Understood, Thanks.

**Review by mrgrain — APPROVED** [review]

**Pull request merged**